diff.py

```diff
@@ -822,12 +828,82 @@ def __hash__(self) -> int:
         return hash(self.key2)
 
 
+class DifferenceNormalizer:
+    def normalize(self, line: Line) -> str:
+        row = line.original
+        row = self._normalize_arch_specific(line, row)
+        row = maybe_normalize_large_imms(row)
```

Comment:
inline maybe_normalize_large_imms here and remove it, since this is the only usage